Tiltfile

```diff
@@ -231,6 +234,9 @@ def enable_providers():
     for name in union_enable_providers:
         enable_provider(name)
 
+def kustomize(path):
+    return str(local("kustomize build {} | {}".format(path, envsubst_cmd), quiet = True))
```

Comment:
WDYT about running `local("make envsubst")` before this so we install envsusbt if it hasn't already been?

Reply:
Just out of curiosity, if this method is being added now what was the tilt script calling before? Or is this overriding some `kustomize(path)` function that's defined elsewhere?

Reply:
I should probably rename the func, tilt provides a function with the same name which does its own handling (that's what was invoked before)

Reply:
Yeah, Tilt's falls back to `kubectl kustomize` if `kustomize` isn't on the PATH, but that fails with our usage of kustomize (as I learned on my newly provisioned machine today)

https://docs.tilt.dev/api.html#api.kustomize

Reply:
Well, TIL. Yeah, you don't have to change the name of the method if you don't want to. Perhaps a commenting explaining above would do just fine. Or both 🤷 🙂